code/modules/reagents/oldchem/reagents/reagents_drugs.dm

```diff
@@ -0,0 +1,95 @@
+/datum/reagent/serotrotium
+	name = "Serotrotium"
+	id = "serotrotium"
+	description = "A chemical compound that promotes concentrated production of the serotonin neurotransmitter in humans."
+	reagent_state = LIQUID
+	color = "#202040" // rgb: 20, 20, 40
+
+/datum/reagent/serotrotium/on_mob_life(var/mob/living/M as mob)
+	if(ishuman(M))
+		if(prob(7)) M.emote(pick("twitch","drool","moan","gasp"))
+		holder.remove_reagent(src.id, 0.25 * REAGENTS_METABOLISM)
+	return
+
+
+/datum/reagent/lithium
+	name = "Lithium"
+	id = "lithium"
+	description = "A chemical element."
+	reagent_state = SOLID
+	color = "#808080" // rgb: 128, 128, 128
+
+/datum/reagent/lithium/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	if(M.canmove && !M.restrained() && istype(M.loc, /turf/space))
+		step(M, pick(cardinal))
+	if(prob(5)) M.emote(pick("twitch","drool","moan"))
+	..()
+	return
+
+
+/datum/reagent/hippies_delight
```

Comment:
That's...weird. I'd definitely recommend making it a drink sub-type if you don't mind.